discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -19,7 +19,7 @@ class TerraformController(LibvirtController):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self.cluster_suffix = self._get_random_name()
-        self.cluster_name = kwargs.get('cluster_name', f'{consts.CLUSTER_PREFIX}') + "-" + self.cluster_suffix
+        self.cluster_name = kwargs.get('cluster_name', f'{consts.CLUSTER_PREFIX}' + "-" + self.cluster_suffix)
```

Comment:
sure? i think cluster_name was always set till now, so suffix was added in any case. It was on purpose

Reply:
here it is made to be sure we have unique vm-name prefix

Reply:
cluster_name is what the user wanted.  Why change it?
Anyway for our purpose, we need to verify that the cluster_name here is the same as known by Assisted Service.  Otherwise, we will have problem with DNS names. For example, api-int.<cluster-name>.<base-domain>

Reply:
the suffix is added so that the vm-names are unique when running in parallel

Reply:
So it should be the user responsibility.  We need to verify that the cluster name known by assisted-service is the same as known by TF.  Otherwise, the DNS names will not be set correctly.